architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -262,11 +328,26 @@ An _image stream tag_ is a named pointer to an image in an _image stream_. It
 is often abbreviated as _istag_. It can reference any local or externally
 managed image. It contains a history of images represented as a stack of all
 images the tag ever pointed to. Whenever a new or existing image is tagged
-under particular _istag_, it is just placed at the first position in the
-history stack. Image previously occupying the top position will be available at
-the second position etc. This allows for easy rollbacks to make tags point to
+under particular image stream tag, it is just placed at the first position in the
+history stack. The image previously occupying the top position will be available at
+the second position, and so forth. This allows for easy rollbacks to make tags point to
 historical images again.
 
+Image stream tags can be _reference tags_ or _version-specific tags_. 
+
+* Version-specific tags point to a particular version of an image, such as Python 3.5.
+
+* Reference tags follow another image stream tag and could be updated in the future to change which image they follow, much like a symlink. Note that these new levels are not guaranteed to be backwards compatible. 
```

Comment:
It's worth mentioning that reference tags are limited only to a *single* image stream, they can't reference other IS.

Reply:
Added a note below:
Reference tags are limited to a single image stream and cannot reference other image streams.